### PR TITLE
Name checks closer to system preferences

### PR DESCRIPTION
**Pull request by dmurko**

### CODE REVIEW

**Comment by dmurko** [comment]
## Rename "Auto login"

In System Preferences, it is called Automatic login and it's "off".

Screenshot 2021-08-02 at 18 00 40

**Comment by dmurko** [comment]
## Rename Firewall

"Firewall is on"

Screenshot 2021-08-02 at 18 05 03

**Comment by dmurko** [comment]
## Screen saver

Screenshot 2021-08-02 at 18 06 52

**Comment by dmurko** [comment]
## Password required

@dz0ny Should this always be "immediately"?


"Password after sleep or screen saver"
Screenshot 2021-08-02 at 18 08 35

**Comment by dmurko** [comment]
## FileVault

"FileVault is on"
Screenshot 2021-08-02 at 18 10 57

**Review by dz0ny — APPROVED** [review]

File: Pareto/Checks/Screensaver.swift [hunk 1]
```diff
@@ -10,7 +10,7 @@ import os.log
 
 class ScreensaverCheck: ParetoCheck {
     final var ID = "13e4dbf1-f87f-4bd9-8a82-f62044f002f4"
-    final var TITLE = "Screen locks in under 5 minutes"
+    final var TITLE = "Screen saver shows in under 5 minutes"
```
Comment (zupo) [inline Pareto/Checks/Screensaver.swift:13]: Disagree on this one. The point is not that the screen saver shows in under 5 minutes. The point is that the screen is locked. For this, you need two things:
* show screensaver OR go to sleep
* lock screen when screensaver is shown or computer goes to sleep

Comment (dmurko) [reply Pareto/Checks/Screensaver.swift:13]: @zupo Unless I'm missing something, the setting for this is literally "show screen saver":

Screenshot 2021-08-02 at 18 06 52

Comment (zupo) [reply Pareto/Checks/Screensaver.swift:13]: You are missing something. Screen saver is only one way to lock the screen. Sleep is the other.

Comment (dmurko) [reply Pareto/Checks/Screensaver.swift:13]: @zupo Please show a screenshot of the setting and where it is.

Comment (zupo) [reply Pareto/Checks/Screensaver.swift:13]: Ah, sorry, here it goes:

Screenshot 2021-08-03 at 10 17 37

Comment (dmurko) [reply Pareto/Checks/Screensaver.swift:13]: It's not the same setting:

Screenshot 2021-08-03 at 10 19 51

So we either need to add another check, or have this as AND in the current check.

Comment (zupo) [reply Pareto/Checks/Screensaver.swift:13]: This check needs to support both approaches. The end result is that the screen locks in 5 minutes. It does not matter whether through screen saver or sleep.

Comment (dmurko) [reply Pareto/Checks/Screensaver.swift:13]: @zupo That is a bug report or new check. As it stands now, the recommended name change is more accurate and reflects the settings better. When we do add the check, we might decide to go with a separate line item, so I would propose we merge this now.

Comment (zupo) [reply Pareto/Checks/Screensaver.swift:13]: The current check is broken. Changing the name will not fix it. Fixing the check will fix it.

Comment (dz0ny) [reply Pareto/Checks/Screensaver.swift:13]: Energy Saver would be a different check.  Bottom line is that any of [Screen Saver, Security, and Energy Saver] can trigger the "requirement to enter password" unconditionally.  And at the moment we are checking just the screen saver which will trigger more frequently.

**Comment by dz0ny** [comment]
> ## Password required
> @dz0ny Should this always be "immediately"?
> 
> "Password after sleep or screen saver"
> Screenshot 2021-08-02 at 18 08 35

I think that for work from home 5min is fine it different thing if you are in a public environment. I would keep it at 5min for now and then add an option to change it.

**Pull request merged**